src/views/WorkspaceDetail.vue

```diff
@@ -143,107 +163,109 @@ const workspaceNameRules: Array<(x: string) => string|boolean> = [
   (x: string) => !surroundingWhitespace.test(x) || 'Workspace name cannot begin or end with whitespace',
 ];
 
-export default Vue.extend({
+export default defineComponent({
   name: 'WorkspaceDetail',
+
   components: {
     TablePanel,
     NetworkPanel,
     WorkspaceOptionMenu,
   },
+
   props: {
     workspace: {
       type: String as PropType<string>,
       required: true,
     },
   },
-  data() {
-    return {
-      localWorkspace: null as string | null,
-      editing: false,
-      requestError: null as string | null,
-      loading: false,
-    };
-  },
 
-  computed: {
-    nodeTables: () => store.getters.nodeTables,
-    edgeTables: () => store.getters.edgeTables,
-    tables: () => store.getters.tables,
-    networks: () => store.getters.networks,
+  setup(props) {
+    // const router = useRouter();
```

Comment:
This is how I use the router with CompositionAPI on Vue2

```suggestion
  setup(props, ctx) {
    const router = ctx.root.$router;
```

Reply:
See c141af6